backends/tests/test_sycl_queue_manager.cpp

```diff
@@ -244,6 +244,22 @@ TEST_F (TestDPPLSyclQueueManager, CheckIsCurrentQueue2)
     DPPLQueueMgr_PopQueue();
 }
 
+TEST_F (TestDPPLSyclQueueManager, CreateQueueFromDeviceAndContext)
+{
+    auto Q = DPPLQueueMgr_GetCurrentQueue();
+    auto D = DPPLQueue_GetDevice(Q);
```

Comment:
Not important here, but all the `DPPL*_Get*()` calls should be followed by a corresponding `Delete` to avoid leaks.